data/core/units/fake/Test_Unit_Require.cfg

```diff
@@ -0,0 +1,24 @@
+#textdomain wesnoth-units
+
+# This unit is used for testing the require traits functionality.
+
+[unit_type]
+	id=Test Unit Require
+	name=_ "dummy_unit^Test Unit Require"
+	race=human
+	image="misc/blank-hex.png"
+	ignore_race_traits=yes
+	{TRAIT_TEST_REQUIRE}
+	{TRAIT_TEST_REQUIRED}
+	hitpoints=1
+	movement_type=fly
+	movement=1
+	experience=1
+	level=1
+	alignment=neutral
+	advances_to=null
+	cost=1
+	usage=scout
+	hide_help=yes
+	do_not_list=yes
+[/unit_type]
```

Comment:
Both of these unit types should also be located in the `data/test/` directory somewhere.

Reply:
I have made a new folder for test units in general. Where do I add them to the environment, the tests can't seem to find them now.

Reply:
For regular units, you'd need to add an include for them in https://github.com/wesnoth/wesnoth/blob/master/data/core/units.cfg

For test-only units, I believe you should be able to include them similarly in https://github.com/wesnoth/wesnoth/blob/master/data/test/_main.cfg

Reply:
Added test unit capability to test textdomain :+1: